wallet-confs/blocktest--v4.3.3.conf

```diff
@@ -0,0 +1,14 @@
+server=1
+listen=1
+rpcuser=
+rpcpassword=
+testnet=1 
+txindex=1
+
+
+
+ 
+[test]
+rpcallowip=0.0.0.0/0
+port=41419
+rpcport=41474
```

Comment:
These two port numbers will get reversed once you update `tblock.base.j2` as suggested above, then rerun `app.py`.